src/datatypes/numeric.js

```diff
@@ -24,7 +24,14 @@ function generateFunctions(bufferReader,bufferWriter,size)
     };
   };
   var writer=function(value, buffer, offset) {
-    buffer[bufferWriter](value, offset);
+    try {
```

Comment:
If you want to add debugging information, use the `debug` function. I still think this one is out-of-place. Might as well add debug info in every single reader.